netlib/http/cookies.py

```diff
@@ -187,13 +202,17 @@ def parse_set_cookie_header(line):
     """
         Parse a Set-Cookie header value
 
-        Returns a (name, value, attrs) tuple, or None, where attrs is an
-        CookieAttrs dict of attributes. No attempt is made to parse attribute
-        values - they are treated purely as strings.
+        Returns a list of (name, value, attrs) tuple for each cokie, or None.
+        Where attrs is a CookieAttrs dict of attributes. No attempt is made
+        to parse attribute values - they are treated purely as strings.
     """
-    pairs = _parse_set_cookie_pairs(line)
-    if pairs:
-        return pairs[0][0], pairs[0][1], CookieAttrs(tuple(x) for x in pairs[1:])
+    cookies = [
+        (pairs[0][0], pairs[0][1], CookieAttrs(tuple(x) for x in pairs[1:]))
+        for pairs in _parse_set_cookie_pairs(line) if pairs
+    ]
+
+    if cookies:
+        return cookies
```

Comment:
This implicitly return None - which a neat way to do it - but I fear a bit too much 😄 
I think we should make it explicit in that case - what do you think?